package.json

```diff
@@ -232,6 +232,10 @@
                 "command": "aks.aksAbortLastOperationInCluster",
                 "title": "Abort Last Operation"
             },
+            {
+                "command": "aks.aksReconcileCluster",
+                "title": "Reconcile Operation"
```

Comment:
Could we call this just 'Reconcile', or maybe 'Reconcile Cluster'?

As it is, it sounds a bit like 'reconcile the operation', which doesn't make sense.